internal/pkg/agent/cmd/watch.go

```diff
@@ -35,14 +34,32 @@ import (
 const (
 	watcherName     = "elastic-agent-watcher"
 	watcherLockFile = "watcher.lock"
+
+	// flag names
+	takedownFlagName      = "takedown"
+	takedownFlagShorthand = "t"
+
+	rollbackFlagName      = "rollback"
+	rollbackFlagShorthand = "r"
+
+	// error exit codes
+	errorSettingParentSignalsExitCode = 6
+	errorRollbackToValue              = 7
+	errorRollbackFailed               = 8
```

Comment:
Do we want to define exit codes 1-5 here as well?

Reply:
We only react to these specific errors in this code path, so defining exit codes 1–5 here isn't necessary. Predefining exit codes that we don't handle would add noise without providing value.
If it's needed we can always fix this in a follow-up PR.